pkg/api/v1/client/testworkflow.go

```diff
@@ -121,6 +122,24 @@ func (c TestWorkflowClient) ExecuteTestWorkflow(name string, request testkube.Te
 	return c.testWorkflowExecutionTransport.Execute(http.MethodPost, uri, body, nil)
 }
 
+// ExecuteTestWorkflows starts new external test workflow executions, reads data and returns IDs
+// Executions are started asynchronously client can check later for results
+func (c TestWorkflowClient) ExecuteTestWorkflows(selector string, concurrencyLevel int, request testkube.TestWorkflowExecutionRequest) (executions []testkube.TestWorkflowExecution, err error) {
+	uri := c.testWorkflowExecutionTransport.GetURI("/test-workflow-executions")
+
+	body, err := json.Marshal(request)
+	if err != nil {
+		return executions, err
+	}
+
+	params := map[string]string{
+		"selector":    selector,
+		"concurrency": strconv.Itoa(concurrencyLevel),
```

Comment:
As above - I don't think that we should have concurrency at this level

Reply:
ok